src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1239,7 +1239,7 @@ public void LangVersion()
             parsedArgs = DefaultParse(new[] { "/langversion:latest", "a.cs" }, _baseDirectory);
             parsedArgs.Errors.Verify();
             Assert.Equal(LanguageVersion.Latest, parsedArgs.ParseOptions.SpecifiedLanguageVersion);
-            Assert.Equal(defaultVersion, parsedArgs.ParseOptions.LanguageVersion);
+            Assert.Equal(LanguageVersion.CSharp7_1, parsedArgs.ParseOptions.LanguageVersion);
```

Comment:
Well   `LanguageVersion.Latest  is literally int.MaxValue   
SpecifiedVersion can be that, but actual languageVersion can't.

I guess, I will just remove the check

---
In reply to: [101640091](https://github.com/dotnet/roslyn/pull/17169#discussion_r101640091) [](ancestors = 101640091)